frontend/package.json

```diff
@@ -3,6 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "productName": "Headlamp",
+  "engines" : {
+    "npm" : ">=10.0.0",
+    "node" : ">=20.9.0"
```

Comment:
This is a nice dx change but i was wandering if this is the limit we have will node 18 not work??

Reply:
We had some people try it at the hackerthon and fail to get the install working with node 18. We also only test in CI with node 20.x.

This version of node is the first LTS release in the 20.x series.